From 7d8b149681d614d19a988a5328cfc7de464feddf Mon Sep 17 00:00:00 2001
From: Jose Santiago <jsantiago@haivision.com>
Date: Thu, 15 Apr 2021 14:04:27 -0500
Subject: [PATCH] Fix Build BSD

This fixes build under BSD. Tested with OpenBSD-6.4 and FreeBSD-11.3.
---
 tests/testutil.hpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tests/testutil.hpp b/tests/testutil.hpp
index c6f5e4de78..3a5402aa16 100644
--- a/tests/testutil.hpp
+++ b/tests/testutil.hpp
@@ -42,6 +42,8 @@
 #if defined _WIN32
 #include "../src/windows.hpp"
 #else
+#include <sys/socket.h>
+#include <netinet/in.h>
 #include <arpa/inet.h>
 #include <unistd.h>
 #include <stdlib.h>
